### PR TITLE
committed libraries to project

### LINKED ISSUE #70: Commit libraries to project
Having to parse long URLs to determine what the project dependencies are isn't great, but the project is also too small to justify the pain of npm. Instead, we should commit our library sources into the repository, then import those.

### PR DESCRIPTION
**Pull request by DanielMwendwa**
closes #70

### CODE REVIEW

**Review by IsaackMwenda — APPROVED** [review]

File: public/coda/coding_progress.html [hunk 1]
```diff
@@ -6,21 +6,18 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <link rel="shortcut icon" type="image/png" href="favicon.png" />
     <title>Data Pipeline Dashboard</title>
-    <!-- Compiled and minified CSS -->
-    <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js" integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNDz0W1" crossorigin="anonymous"></script>
```
Comment (IsaackMwenda) [inline public/coda/coding_progress.html:11]: Are we still using popper.min.js? if yes import it too.

File: public/media_engagement/imaqal_sms_traffic.html [hunk 2]
```diff
@@ -5,21 +5,17 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <link rel="shortcut icon" type="image/png" href="../../public/coda/favicon.png" />
     <title>Engagement & Media Dashboard</title>
-    <!-- Compiled and minified CSS -->
-    <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js" integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNDz0W1" crossorigin="anonymous"></script>
```
Comment (IsaackMwenda) [inline public/media_engagement/imaqal_sms_traffic.html:10]: ditto

File: public/media_engagement/worldbank_plr_sms_traffic.html [hunk 3]
```diff
@@ -5,21 +5,17 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <link rel="shortcut icon" type="image/png" href="../../public/coda/favicon.png" />
     <title>Engagement & Media Dashboard</title>
-    <!-- Compiled and minified CSS -->
-    <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js" integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNDz0W1" crossorigin="anonymous"></script>
```
Comment (IsaackMwenda) [inline public/media_engagement/worldbank_plr_sms_traffic.html:10]: ditto

**Review by IsaackMwenda — CHANGES_REQUESTED** [review]

File: public/media_engagement/imaqal_sms_traffic.html (hunk 2 above)
Comment (DanielMwendwa) [reply public/media_engagement/imaqal_sms_traffic.html:10]: The minified version of bootstrap contains popper js

Comment (IsaackMwenda) [reply public/media_engagement/imaqal_sms_traffic.html:10]: great thanks

**Review by IsaackMwenda — APPROVED** [review]

**Review by lukechurch — CHANGES_REQUESTED** [review]
What licence are these projects under? I think we need to reference them in a third party imports readme file, we're not allowed to relicenence them under MIT by virtual of including them in the repo. Please chat with @IsaackMwenda if unsure about this.

File: open-source-license-acknowledgements-and-third-party-copyrights.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
+# Open Source License Acknowledgements and Third-Party Copyrights
+
+Operations-Dashboard is licensed under the terms of the [MIT License](./LICENSE).
```
Comment (IsaackMwenda) [inline open-source-license-acknowledgements-and-third-party-copyrights.md]: link the License file here

File: open-source-license-acknowledgements-and-third-party-copyrights.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
+# Open Source License Acknowledgements and Third-Party Copyrights
```
Comment (IsaackMwenda) [inline open-source-license-acknowledgements-and-third-party-copyrights.md]:
```suggestion
# Open Source and Third-Party Softwares Licenses and Copyrights Acknowledgements.
```

File: open-source-license-acknowledgements-and-third-party-copyrights.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+Copyright (c) 2011-2019 Twitter, Inc.<br/> 
+Copyright (c) 2011-2019 The Bootstrap Authors
+
+### Font awesome
+License: Font Awesome Free License<br/> 
+https://github.com/FortAwesome/Font-Awesome/blob/master/LICENSE.txt
+
+## Sources of information about licenses
+The GitHub repositories of these packages are used as the source of the information about licenses.
+
```
Comment (IsaackMwenda) [inline open-source-license-acknowledgements-and-third-party-copyrights.md:36]: remove unnecessary empty lines

File: open-source-license-acknowledgements-and-third-party-copyrights.md (hunk 5 above)
Comment (IsaackMwenda) [reply open-source-license-acknowledgements-and-third-party-copyrights.md]: Rename the file name too and capitalize it

**Review by IsaackMwenda — CHANGES_REQUESTED** [review]
Thanks @DanielMwendwa . Requesting changes due to minor comments.

File: open-source-license-acknowledgements-and-third-party-copyrights.md (hunk 4 above)
Comment (DanielMwendwa) [reply open-source-license-acknowledgements-and-third-party-copyrights.md]: the link redirects to this file https://github.com/AfricasVoices/Operations-Dashboard/blob/Commit_libraries_to_project/LICENSE

File: open-source-license-acknowledgements-and-third-party-copyrights.md (hunk 6 above)
Comment (DanielMwendwa) [reply open-source-license-acknowledgements-and-third-party-copyrights.md:36]: I have removed the lines. Also note that headings have spacing around them by default.

**Review by IsaackMwenda — APPROVED** [review]
This looks okay now. Thanks @DanielMwendwa .

**Review by lukechurch — APPROVED** [review]

**Pull request merged**